system-addon/lib/SectionsManager.jsm

```diff
@@ -52,6 +52,10 @@ const BUILT_IN_SECTIONS = {
     title: {id: "header_highlights"},
     maxRows: 3,
     availableContextMenuOptions: ["CheckBookmark", "SaveToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl", "DeleteUrl"],
+    infoOption: {
```

Comment:
hmm, it is here. it isn't showing up? /me tests

Reply:
WFM 😕 

<img width="516" alt="screen shot 2017-09-29 at 9 11 09 am" src="https://user-images.githubusercontent.com/36629/31017357-2ba1c840-a4f6-11e7-9af8-ca8bfe27d53d.png">

Reply:
Oh oops. I think I ran without `./mach build browser/app` to update the lib files. The content files updated. Doh!

Reply:
Yup WFM now too.